applications/GeoMechanicsApplication/tests/test_k0_procedure_process.py

```diff
@@ -214,6 +215,60 @@ def test_k0_procedure_k0_nc_ocr(self):
         sig_xy = sig_integrationpoint1_element1[0,1]
         self.assertEqual( sig_xy, 0.0 )
 
+    def test_k0_procedure_k0_nc_ocr_field(self):
+        """
+        Test to check if CAUCHY_STRESS_XX is correctly derived from CAUCHY_STRESS_YY using K0_NC and
+        an OCR supplied from a set_parameter_field_proces
+        """
+
+        test_name = os.path.join("test_k0_procedure_process", "test_k0_procedure_k0_nc_ocr_field")
+        file_path = test_helper.get_file_path(test_name)
+
+        # run simulation
+        simulation = test_helper.run_kratos(file_path)
+
+        # retrieve Cauchy stress tensor
+        cauchy_stresses = test_helper.get_on_integration_points(simulation,Kratos.CAUCHY_STRESS_TENSOR)
+
+        # compare cauchy_stress_xx = k0 * cauchy_stress_yy, cauchy_stress_xy = 0.
```

Comment:
This is probably because of my OCD...
```suggestion
        # compare cauchy_stress_xx = k0 * cauchy_stress_yy, cauchy_stress_xy = 0.0
```

Reply:
Done.